fragmentstest/src/main/res/values/dialog_fragment_test_theme.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.DialogFragmentTest" parent="Theme.MaterialComponents">
+        <!--
+        For some reason the default value of `alertDialogTheme` (which is
+        `ThemeOverlay.MaterialComponents.Dialog.Alert`  ) will cause `launchFragment` to
+        explode. This is most likely a bug in FragmentScenario (or Robolectric itself)
```

Comment:
This is pretty grim. I'm not even sure where to file a bug. I'll take all this stuff and play around with it sometime soon so I can understand which problems we're having with Robolectric are holes in documentation, which are bugs in androidx test and which are bugs in Robolectric itself.

Reply:
You have 2 spaces before `)`